__tests__/get-todo-config-test.ts

```diff
@@ -0,0 +1,168 @@
+import { unlink } from 'fs-extra';
+import { join } from 'path';
+import { getTodoConfig } from '../src';
+import { FakeProject } from './__utils__/fake-project';
+
+describe('get-todo-config', () => {
+  let project: FakeProject;
+
+  beforeEach(() => {
+    project = new FakeProject();
+    project.writeSync();
+  });
+
+  afterEach(() => {
+    project.dispose();
+    delete process.env['TODO_DAYS_TO_WARN'];
+    delete process.env['TODO_DAYS_TO_ERROR'];
```

Comment:
Probably don't need to brackets + quotes. Not a big deal though.

Reply:
Ya, I hate seeing snake cased props, so it 'feels' better to me to be strings. 🤷‍♂️